README.md

```diff
@@ -467,57 +299,86 @@ pg_dump:
 PGHOST
```

Comment:
Maybe format the environment variables in the same way as they are formatted in the following section (1.2) i.e. list them within the paragraph, using the same phrasing and formatting.